source/adapters/level_zero/common.cpp

```diff
@@ -88,7 +88,9 @@ bool setEnvVar(const char *name, const char *value) {
 ZeUSMImportExtension ZeUSMImport;
 
 // This will count the calls to Level-Zero
+#ifndef UR_L0_CALL_COUNT_IN_TESTS
 std::map<std::string, int> *ZeCallCount = nullptr;
```

Comment:
I really don't like this. #1454 is meant to give programmatic access to internals like this in tests. So please add a TODO.